libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -626,8 +625,9 @@ dictionary BitcoinAddressData {
 };
 
 dictionary LnUrlPaySuccessData {
-    SuccessActionProcessed? success_action;
     Payment payment;
+    Optional<SuccessActionProcessed> success_action;
```

Comment:
The success_action field doesn't need a change here? Does the order matter?
At least it should be `SuccessActionProcessed? success_action;`